app/utils/query_helpers.py

```diff
@@ -156,11 +165,16 @@ def get_available_stats_for_domain(domain: type[DeclarativeBase]) -> list[str]:
 
 
 @functools.cache
-def compute_stat_offerings() -> list[StatOffering]:
+def compute_stat_offerings(game: str) -> list[StatOffering]:
```

Comment:
Instead of having a conditional here, I think it would be better to make a separate function for UYA so the code is simpler.